version.php

```diff
@@ -28,9 +28,9 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version  = 2023081601;
+$plugin->version  = 2023082301;
 $plugin->requires = 2020061500;
-$plugin->release = '3.1.0 (Build: 2023081601)';
+$plugin->release = '3.1.0 (Build: 2023082301)';
```

Comment:
@rohansharmasitoula Update the build version to 3.1.1 and give the PR to a release branch

Reply:
done